### ISSUE TITLE
Internal puppet "environment" is overriden

### ISSUE THREAD

**Issue by Vincent--** [voxpupuli/puppet-php#268]
This module breaks all my puppet configuration since the last commits from https://github.com/voxpupuli/puppet-php/commit/2a02669b8dc8b53376409e9b61500ba025b47695

I got this error message:
```
Error: Could not find a directory environment named 'undef' anywhere in the path: /foo/bar. Does the directory exist? on node devvm-xxxx
```

**Comment by alexjfisher** [comment]
There's an open PR that fixes this https://github.com/voxpupuli/puppet-php/pull/274

**Comment by juniorsysadmin** [comment]
Fixed by #274 I think, closing
